# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-backgrounds/animations/background-color-transparent-animation-in-body.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/css-backgrounds/animations/background-color-transparent-animation-in-body.html.ini
--- a/testing/web-platform/meta/css/css-backgrounds/animations/background-color-transparent-animation-in-body.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/css-backgrounds/animations/background-color-transparent-animation-in-body.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,8 +1,14 @@
 [background-color-transparent-animation-in-body.html]
   expected:
-    if (os == "win") and debug and webrender and not swgl: PASS
-    if (os == "linux") and not fission and debug and not webrender: PASS
+    if (os == "win") and webrender and not fission and debug and not swgl: PASS
+    if (os == "win") and webrender and not fission and debug and swgl: ["PASS", "TIMEOUT"]
+    if (os == "linux") and webrender and not debug and not fission: ["PASS", "TIMEOUT"]
+    if (os == "win") and not webrender and not debug and (processor == "x86"): ["PASS", "TIMEOUT"]
+    if (os == "win") and webrender and not fission and not debug: ["PASS", "TIMEOUT"]
+    if (os == "android") and not debug and webrender: ["PASS", "TIMEOUT"]
     if (os == "android") and not debug and not webrender: PASS
-    if (os == "linux") and fission and not debug: ["TIMEOUT", "PASS"]
+    if (os == "linux") and webrender and debug: ["PASS", "TIMEOUT"]
+    if (os == "linux") and not webrender and debug: PASS
+    if (os == "win") and webrender and fission: ["PASS", "TIMEOUT"]
     if (os == "android") and debug: PASS
-    [PASS, TIMEOUT]
+    [TIMEOUT, PASS]